java/java.hints/nbproject/project.xml

```diff
@@ -278,6 +278,24 @@
                         <specification-version>1.25</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
```

Comment:
Having java.hints depend on all these project modules does not seem right - if e.g. project.ant is disabled, should java.hints be disabled as well? To me, does not seem so. I'd suggest to:
-place maven handling to maven.hints (duplicate the ErrorRule there)
-place the ant handling either to a separate module; or avoid any hard dependency on project.ant, etc. here (after all, the nbproject/project.properties should be readable without the dependency).